data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv

```diff
@@ -0,0 +1 @@
+[Header],,,,,,,,,,,
IEMFileVersion,5,,,,,,,,,,
Date,08/21/19,,,,,,,,,,
Workflow,GenerateFASTQ,,,,,,,,,,
Application,HiSeq FASTQ Only,,,,,,,,,,
Instrument Type,HiSeq 1500/2500,,,,,,,,,,
Assay,Agilent_Sure_SelectXT,,,,,,,,,,
Index Adapters,Agilent_Sure_SelectXT,,,,,,,,,,
Description,PE-Amp-Lin-Block-Hyb-v 8.0,,,,,,,,,,
Chemistry,default,,,,,,,,,,
Operator,Daniel Scholtyssik / Katayoun Alemazkur ,,,,,,,,,,
Investigator,Layal Yasin / Ute Fischer,,,,,,,,,,
Backup_NAS_name,Ingerimm,,,,,,,,,,
kind_of_run,2x101bp,,,,,,,,,,
[Reads],,,,,,,,,,,
101,,,,,,,,,,,
101,,,,,,,,,,,
,,,,,,,,,,,
[Settings],,,,,,,,,,,
ReverseComplement,0,,,,,,,,,,
Adapter,AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,,,,,,,,,,
AdapterRead2,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT,,,,,,,,,,
,,,,,,,,,,,
[Data],,,,,,,,,,,
Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,Sample_Project,Description,spike_entity_id,spike_entity_role,spike_ignore_sample
1,KB0274_f,,,,A01,ATGCCTAA,Keimbahn,set1,KB0274,father,
1,KB0274_m,,,,B01,GAATCTGA,Keimbahn,set1,KB0274,mother,
1,KB0272_c,,,,C01,AACGTGAT,Keimbahn,set1,KB0272,patient,
1,KB0272_f,,,,D01,CACTTCGA,Keimbahn,set1,KB0272,father,
2,KB0274_f,,,,A01,ATGCCTAA,Keimbahn,set1,KB0274,father,
2,KB0274_m,,,,B01,GAATCTGA,Keimbahn,set1,KB0274,mother,
2,KB0272_c,,,,C01,AACGTGAT,Keimbahn,set1,KB0272,patient,
2,KB0272_f,,,,D01,CACTTCGA,Keimbahn,set1,KB0272,father,
3,KB0272_m,,,,E01,GCCAAGAC,Keimbahn,set2,KB0272,mother,
3,KB0232_c,,,,F01,GACTAGTA,Keimbahn,set2,KB0232,patient,
3,KB0232_f,,,,G01,ATTGGCTC,Keimbahn,set2,KB0232,father,
3,KB0232_m,,,,H01,GATGAATC,Keimbahn,set2,KB0232,mother,
4,KB0272_m,,,,E01,GCCAAGAC,Keimbahn,set2,KB0272,mother,
4,KB0232_c,,,,F01,GACTAGTA,Keimbahn,set2,KB0232,patient,
4,KB0232_f,,,,G01,ATTGGCTC,Keimbahn,set2,KB0232,father,
4,KB0232_m,,,,H01,GATGAATC,Keimbahn,set2,KB0232,mother,
5,KB0266_c,,,,A01,ATGCCTAA,Keimbahn,set3,KB0266,patient,
5,KB0266_f,,,,B01,GAATCTGA,Keimbahn,set3,KB0266,father,
5,KB0266_m,,,,C01,AACGTGAT,Keimbahn,set3,KB0266,mother,
5,KB0270_c,,,,D01,CACTTCGA,Keimbahn,set3,KB0270,patient,
6,KB0266_c,,,,A01,ATGCCTAA,Keimbahn,set3,KB0266,patient,
6,KB0266_f,,,,B01,GAATCTGA,Keimbahn,set3,KB0266,father,
6,KB0266_m,,,,C01,AACGTGAT,Keimbahn,set3,KB0266,mother,
6,KB0270_c,,,,D01,CACTTCGA,Keimbahn,set3,KB0270,patient,
7,KB0270_f,,,,E01,GCCAAGAC,Keimbahn,set4,KB0270,father,
7,KB0270_m,,,,F01,GACTAGTA,Keimbahn,set4,KB0270,mother,
7,KB0275_c,,,,G01,ATTGGCTC,Keimbahn,set4,KB0275,patient,
7,KB0275_f,,,,H01,GATGAATC,Keimbahn,set4,KB0275,father,
8,KB0270_f,,,,E01,GCCAAGAC,Keimbahn,set4,KB0270,father,
8,KB0270_m,,,,F01,GACTAGTA,Keimbahn,set4,KB0270,mother,
8,KB0275_c,,,,G01,ATTGGCTC,Keimbahn,set4,KB0275,patient,
8,KB0275_f,,,,H01,GATGAATC,Keimbahn,set4,KB0275,father,
```

Comment:
looks like you messed up the line breaks. Everything is in one single line?!

Reply:
It seems to be an issue in github. Raw file looks right. I am starting my review now...

Reply:
shall I create or add a (line breaks) to  the sample sheet